js/songs/_song_form.html

```diff
@@ -0,0 +1,16 @@
+<form>
+  <input name = 'title' placeholder="Title" data-ng-model='song.title'/>
```

Comment:
Lots of folks in this class have been putting spaces around the `=` in HTML attributes. [My thoughts](http://stackoverflow.com/a/3551614/2053389)